app/views/assessments/_results_panel.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="result-summary">
+  <div class="result-pane result-fixed">
+    <div class="feedback-header">
+      <h3>Results</h3>
+    </div>
+    <div id="result-body">
+      <% if !@scoreHash.nil? %>
+        <h4>Autograder Scores</h4>
+        <table class="ui unstackable table">
+          <tbody>
+          <% @scoreHash.each do |key, value| %>
```

Comment:
Nit: Could we do something like `@scoreHash.except("_total")` and remove the `key != '_total'` check below